### PR TITLE
adding administrative levels dataset

### PR DESCRIPTION
**Pull request by maqzi**

### CODE REVIEW

**Review by heikomuller — APPROVED** [review]

File: data/index.json [hunk 1]
```diff
@@ -130,6 +130,35 @@
                     "delim": "\t"
                 }
             }
+        },
+        {
+            "id": "admins",
+            "name": "Geo Administrative Levels",
+            "url": "https://osf.io/y3q8e/download",
+            "checksum": "4a1fd5f3891eaf5deffa3c69c8c18fc3b254847be327ec6961b4a13a180ae2ef",
+            "webpage": "https://osf.io/y3q8e/",
```
Comment (remram44) [inline data/index.json:139]: https://gitlab.com/ViDA-NYU/auctus/datamart-geo is probably a better link, I am only using OSF for storage and there are no details there

**Comment by remram44** [comment]
I am changing `datamart-geo` to accept gzipped files, which will need to be uncompressed after download. Will that work for openclean? My plan was for the hash to still be over the uncompressed data.

**Comment by maqzi** [comment]
Yeah, the openclean reference data repository [supports](https://github.com/VIDA-NYU/reference-data-repository/blob/397461f2e8406d7858c8880306de98c28e801dac/refdata/dataset/base.py#L163) gzip decompression so we should be fine there.

**Comment by heikomuller** [comment]
I think the problem currently is that the hash in refdata is computed over the downloaded file, not the uncompressed file. We would need to add the option to automatically decompress the file after download. The question is whether we want to do this automatically for all compressed files or only for some (e.g., based on a flag in the file description metadata)?

**Comment by remram44** [comment]
@maqzi: This seems to only support text data? I see CSV and JSON, and `gzip.open(..., 'rt')`

I can change it to the hash of the compressed file in datamart-geo.

**Comment by remram44** [comment]
datamart-geo ~0.2.2~ 0.2.3 is released with support for gzipped data. The `info.json` will contain the sha1 of compressed data for compatibility with your code.

I probably won't actually put a gzipped file in there until I am done integrating OpenStreetMap data though.

**Pull request merged**